docs/developer/soffits/minimal_soffit.md

```diff
@@ -0,0 +1,71 @@
+
+# Minimal Soffit
+
+A soffit's interaction with the portal is HTTP-based.  It is posible to write a
+soffit in any language or platform that can accept, process, and respond to a
+connection over HTTP.  At the present time, the creators of Soffit expect to
+develop soffits mostly with [Java][] and [Spring Boot][].
+
+## Minimal Soffit Setup Instructions Using Spring Boot
+
+1. Use the [Spring Initializer][] to create a new Spring Boot project with the
+   following settings:
+    * Gradle Project (recommended)
+    * Packaging=*War* (recommended)
+    * Dependencies=*Cache* (recommended) & *Web* (required)
+    * Additional dependencies you intend to use (optional -- you can add them
+      later)
+1. Add Soffit as a dependency to your project (see below)
```

Comment:
I vote "no" for this one.  My sense (before researching) was that it's a bad idea to use actual numbers because...

* It requires human maintenance;  if you reorder or add one in the middle, you're obligated to re-do them all
* It's confusing;  do the 1, 2, 3, 4 appear in the rendered document as plain text?  Or is it markdown syntax?  (With 1., 1., 1... it's clearer that Markdown manages the numbering)

But since you asked, I also did a little googling to see what others thought...

[Markdown Style Guide](http://www.cirosantilli.com/markdown-style-guide/#unordered):

    Prefer lists only with the marker 1. for ordered lists, unless you intend to refer to items by their number in the same markdown file or externally.

That's just one source, but it's the first hit for "Markdown Style Guide."